### ISSUE TITLE
activity feed could use gene/variant context

### ISSUE THREAD

**Issue by malachig** [griffithlab/civic-v2#461]
In civic v1, always shows the gene/variant names.  

In the V2 curation timeline: https://civicdb.org/curation/activity/curation-timeline

We only see the event type and the entity IDs. Browsing through its hard to find my recent activity, whereas if the variant and gene name were shown I could more readily parse through the recent activity.

**Comment by susannasiebert** [comment]
I'm not quite sure I understand what you are seeing. When I go to the curation feed, it shows the variant names for me. 

<img width="1611" alt="Screen Shot 2022-04-15 at 8 46 32 AM" src="https://user-images.githubusercontent.com/6617192/163578531-217d3837-6d38-41fd-9a36-111fd87f46e5.png">

Are you requesting for the variant tags to include the gene name in addition to the variant name?

**Comment by malachig** [comment]
Yeah, somehow I was not seeing that variant info.  But it would be good to have the gene as well as in V1.  To help with generic variant names like "mutation", "over-expression", etc.

**Comment by susannasiebert** [comment]
Do you think we should include the gene name in all variant tags, irregardless of context? Or is that just an issue in the activity feed?

**Comment by susannasiebert** [comment]
Also, you might've been looking at the event feed for a specific gene/variant/other entity. We disable these "affected entity" tags in this context since you are already on the affected gene/variant/etc page. E.g. https://civicdb.org/variants/12/events. If this is confusing, we can put them back.

**Comment by susannasiebert** [comment]
We should investigate adding the gene & variant tags to the evidence & assertion tags for subjects/originating objects. This might not be easily possible/performant with our current queries.
